src/js/views/maps/ToolbarView.js

```diff
@@ -197,6 +199,30 @@ define([
             return model.get("showNavHelp") || model.get("showFeedback");
           },
         },
+        {
+          label: "Share",
+          icon: "link",
+          action(view) {
+            if (view.$el.find(`.${view.classes.shareUrlView}`).length) return;
+
+            const title = this.linkEl.querySelector(
+              `.${view.classes.linkTitle}`,
+            );
+            const shareUrlView = new ShareUrlView({
+              top: this.linkEl.offsetTop,
+              left: this.linkEl.offsetLeft,
+              linkTitle: title,
+              linkTitleHiddenClassName: view.classes.linkTitleHidden,
+            });
+            shareUrlView.render();
+            // Make sure link's tooltip is hidden while its popup is visible.
+            title.classList.add(view.classes.linkTitleHidden);
```

Comment:
To hide yes, but the visibility is controlled by CSS, so to show it again .show() doesn't work since it will apply an inline style and then become permanently visible. The opposite would be to remove the inline display style attribute. But I think toggling a class is simpler for that